src/test/java/unit/GcsBucketControlled.java

```diff
@@ -35,7 +35,7 @@ void listDescribeReflectCreate() throws IOException {
     // `terra workspace set --id=$id`
```

Comment:
Out of interest, why drop `--format=json` here?

Reply:
I only need the `--format=json` flag when I'm parsing the result (i.e. calling `TestCommand.runAndParseCommandExpectSuccess`) into an object. In the two places below, I only need to make sure the command ran successfully so I can use the default `--format=text` flag instead.